src/pages/settings/Profile/ProfilePage.js

```diff
@@ -74,28 +74,17 @@ class ProfilePage extends Component {
             pronouns,
             timezone = {},
         } = props.myPersonalDetails;
-        const pronounsList = Object.values(this.props.translate('pronouns'));
-
-        let currentUserPronouns = pronouns;
-        let initialSelfSelectedPronouns = '';
-
-        // This handles populating the self-selected pronouns in the form
-        if (pronouns && !pronounsList.includes(pronouns)) {
-            currentUserPronouns = this.props.translate('pronouns.selfSelect');
-            initialSelfSelectedPronouns = pronouns;
-        }
 
         this.state = {
             firstName,
             lastName,
-            pronouns: currentUserPronouns,
-            selfSelectedPronouns: initialSelfSelectedPronouns,
+            pronouns,
+            selfSelectedPronouns: pronouns && !pronouns.startsWith(CONST.PRONOUNS.PREFIX),
```

Comment:
Since `selfSelectedPronouns` is now a boolean (not an array) I think we should change the variable name to indicate it's a boolean value. Maybe `hasSelfSelectedPronouns`?

Reply:
Done!